css/crm-menubar.css

```diff
@@ -205,13 +205,32 @@ body.crm-menubar-over-cms-menu #crm-menubar-toggle-position a i {
      collapsible to desktop (navbar + dropdowns)
   -----------------------------------------------*/
   /* start... (it's not recommended editing these rules) */
-  #civicrm-menu ul{position:absolute;width:12em;}
-  #civicrm-menu li{float:left;}
-  #civicrm-menu.sm-rtl li{float:right;}
-  #civicrm-menu ul li,#civicrm-menu.sm-rtl ul li,#civicrm-menu.sm-vertical li{float:none;}
-  #civicrm-menu a{white-space:nowrap;}
-  #civicrm-menu ul a,#civicrm-menu.sm-vertical a{white-space:normal;}
-  #civicrm-menu .sm-nowrap > li > a,#civicrm-menu .sm-nowrap > li > :not(ul) a{white-space:nowrap;}
+  #civicrm-menu ul {
+    position: absolute;
+    width: 12em;
+}
+  #civicrm-menu li {
+    float: left;
+}
```

Comment:
These `}`s look misaligned...  sometimes phpcbf makes a poor decision, but you can edit by hand, and `phpcs` will accept the better version.

Reply:
OK I fixed those

Reply:
Great. Re-read - and agree all changes look safe+consistent.